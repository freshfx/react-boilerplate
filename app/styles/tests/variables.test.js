import {
  BLACK,
  mediaSize,
  WHITE
} from '../variables'

describe('styles variables', () => {
  it('should return black', () => {
    expect(BLACK).toEqual('#000')
  })

  it('should return white', () => {
    expect(WHITE).toEqual('#fff')
  })

  describe('mediaSize', () => {
    it('should match snapshot', () => {
      expect(() => mediaSize`
        font-size: 13px;
        letter-spacing: 0.5px;
        margin: 0;
      `).toMatchSnapshot()
    })

    it('should not throw if called without any arguments', () => {
      expect(() => mediaSize()).not.toThrow()
    })
  })
})
