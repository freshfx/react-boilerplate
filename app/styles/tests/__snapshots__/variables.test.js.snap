// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles variables mediaSize should match snapshot 1`] = `" font-size: 0.8125rem; letter-spacing: 0.03125rem; margin: 0;"`;
